src/+replab/+str/Table.m

```diff
@@ -0,0 +1,365 @@
+classdef Table < replab.Str
+% Helper to display tables on the REPL
+%
+% >>> replab.str.Table(cell(4,4), 2:3, 'llcc') % unreadable
+
+    properties
+        rowSep % cell(1,nRows+1) of charstring: Row separators
+        colSep % cell(1,nCols+1) of charstring: Column separators
+               %
+               %                                This includes a separator on the left of the first column and
+               %                                on the right of the last column.
+        colAlign % char(1,nCols): Array of 'l' 'c' 'r' (default to 'c')
+        omitRange % integer(1,\*): Range of columns that can be omitted if space is lacking
```

Comment:
Do you need this range to be contiguous, or monotonically increasing, or can it be whatever?